utils.rollup.config.js

```diff
@@ -0,0 +1,39 @@
+import typescript from 'rollup-plugin-typescript2';
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from 'rollup-plugin-commonjs';
+import json from '@rollup/plugin-json';
+import externals from 'rollup-plugin-node-externals';
+
+/**
+ * Copyright 2020 Bonitasoft S.A.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const plugins = [
+  typescript({
+    typescript: require('typescript'),
+  }),
+  externals(),
```

Comment:
:question: for my understanding, can you explain briefly why we need this new plugin here?
I got a warning when building with node 12, is that related?
```
(!) Unused external imports
mkdirSync,copyFileSync,readdirSync imported from external module 'fs' but never used
```

Reply:
I have fixed that in the last commit - it was the problem coming from file-helper